src/cargo/util/toml/mod.rs

```diff
@@ -1423,6 +1444,13 @@ impl TomlManifest {
                 StringOrBool::Bool(_) => {}
             }
         }
+        if let Some(_) = &package.build {
```

Comment:
Could we do a cleanup for the old commit if we're not using them anymore?

Reply:
oh, I forgot